lib/Core/MemoryManager.cpp

```diff
@@ -113,7 +113,7 @@ MemoryObject *MemoryManager::allocate(uint64_t size, bool isLocal,
     return 0;
   }
 
-  uint64_t address = 0;
+  std::uintptr_t address = 0;
```

Comment:
I'm not sure if that makes sense. Let's say we extend KLEE to run 32bit programs on 64bit host machines or vice versa, than `address` is propagated to the client code. Why should it be dependent on the pointer size of the host machine?

Reply:
We discussed this in the past, and the changes required would be significant.  Also, I don't think we want to support KLEE running on a 32-bit machine.  We have decided to drop support a while ago, and I don't think it's worth the engineering effort to change our mind now, is it?  While I know some users to be interested in analysing 32-bit applications, nobody mentioned to me that they'd like to run KLEE on a 32-bit platform.

Reply:
Nevertheless, it's still host specific - even for 32bit on 64bit host.

Reply:
It would be fine given these assumptions, but I agree it's not worth the trouble,